samples: cellular: nrf_cloud_multi_service: add cfg handling (#15147)
nrfconnect/sdk-nrf: samples/cellular/nrf_cloud_multi_service/README.rst

Test counter
============

Every time a sensor sample is sent, this counter is incremented by one, and its current value is sent to `nRF Cloud`_.
A plot of the value of the counter over time is automatically shown in the nRF Cloud portal.
This plot is useful for tracking, visualizing, and debugging connection loss, resets, and re-establishment behavior.

> The test counter can be enabled or disabled using the device shadow.

Reviewer: ```suggestion
You can enable or disable the test counter using the device shadow.
```

In the desired config section, set ``"counterEnable"`` to ``true`` to enable or ``false`` to disable the test counter.

.. code-block:: json

   "desired": {
      "config": {
         "counterEnable": true
      }
   }

> The shadow update can be performed by using the ``View Config`` button on the device page in the nRF Cloud portal or the ``UpdateDeviceState`` endpoint in the `nRF Cloud Rest API`_.
> To ignore the shadow setting so that the test counter is always active, enable the :ref:`CONFIG_TEST_COUNTER <CONFIG_TEST_COUNTER>` option.

Reviewer: ```suggestion
You can perform the shadow update by clicking the :guilabel:`View Config` button on the **Device** page in the nRF Cloud portal or through the ``UpdateDeviceState`` endpoint in the `nRF Cloud Rest API`_.
To ignore the shadow setting so that the test counter is always active, enable the :ref:`CONFIG_TEST_COUNTER <CONFIG_TEST_COUNTER>` Kconfig option.
```

CONFIG_TEST_COUNTER - Enable test counter

>    Enables the test counter.

Reviewer: ```suggestion
   Enable the test counter.
```
